### PR TITLE
Create mediatek contact-us form

### PR DESCRIPTION
**Pull request by petesfrench**
## Done
*In searching around for how static forms were handled in other places, i stumbled across a number of other smaller issue that i have covered and detailed below*

- Create mediatek contact-us form based on [the copydoc](https://docs.google.com/document/d/1WnWVmVuqd1G_BCA060K1LozS2xYFCDli90sGc1SU7ug/edit#heading=h.dxn10eelvkoc)
- Create js function to parse the extra fields and add them to a hidded field 'Comments_from_lead_c'. This is already being done in dynamic-form.js but only applies to interactive forms. And include a js to handle 'Other' check boxes, an example of the HTML needed is found below
- Removes code from [/embedding/index.html](https://github.com/canonical/ubuntu.com/pull/12612/files#diff-56ddfc54a339d2672c4a98dfe78fc989814a78f58941b9ea42156184b27375a8) that didn't appear to do anything.
- Removed some random console.logs
- White space formatting on a file I check out
- Removes unused fips beta form

HTML for other checkbox example:
```
 <div class="js-other-container">
    <label class="p-checkbox">
      <input class="p-checkbox__input js-other-container__other-toggle" type="checkbox" aria-labelledby="priority-workloads-other">
      <span class="p-checkbox__label" id="priority-workloads-other">Other</span>
    </label>
    <input type="text" id="priority-workloads-other-specified" class="js-other-container__input" placeholder="Please specify" style="opacity: 0; margin-top: .25rem;" aria-label="Other architecture">
  </div>
```
## QA

- Go to https://ubuntu-com-12612.demos.haus/download/mediatek/contact-us and sewe that the form matches the copydoc
- Fill in the form and check the payload contains all information
- Check the formid is correct
- Check it returns to a generic thank-you page
- Check the form is recieved on marketo

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-2250

### CODE REVIEW

**Comment by petesfrench** [comment]
@tbille This one is ready for review!

**Comment by tbille** [comment]
@petesfrench is this a known issue and should it be part of this PR?
89RLh0Y6MW

File: templates/download/mediatek/contact-us.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+{% block title %}Contact us{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1WnWVmVuqd1G_BCA060K1LozS2xYFCDli90sGc1SU7ug/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+
+<section class="p-strip--light u-no-padding--bottom">
+  <div class="row">
+    <div class="col-8">
+      <h1>Interested inoptimised Ubuntu on MediaTek's Genio 1200 SoC?</h1>
```
Comment (tbille) [inline templates/download/mediatek/contact-us.html:12]:
```suggestion
      <h1>Interested in optimised Ubuntu on MediaTek's Genio 1200 SoC?</h1>
```
space missing

**Comment by mtruj013** [comment]
@petesfrench I'm afraid the code is still doing something. If you go to the embedding page and check the payload when you submit the form the "tell us more about your needs" field no longer gets passed to `comments_from_lead_c`

On demo: 
Screenshot from 2023-02-28 18-50-29

Live site: 
Screenshot from 2023-02-28 18-52-21

File: templates/download/mediatek/contact-us.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+            <span class="p-radio__label" id="when-do-you-ship-next-3-6-months">Next 3-6 months</span>
+          </label>
+          <label class="p-radio">
+            <input class="p-radio__input" type="radio" aria-labelledby="when-do-you-ship-next-6-12-months" name="when-do-you-ship" value="Next 6-12 months">
+            <span class="p-radio__label" id="when-do-you-ship-next-6-12-months">Next 6-12 months</span>
+          </label>
+          <label class="p-radio u-sv3">
+            <input class="p-radio__input" type="radio" aria-labelledby="when-do-you-ship-no-timeline" name="when-do-you-ship" value="No timeline - just researching">
+            <span class="p-radio__label" id="when-do-you-ship-no-timeline">No timeline - just researching</span>
+          </label>
```
Comment (tbille) [inline templates/download/mediatek/contact-us.html:109]: This section is not wrapped by a `<div class="row">` like the others. It this intended?

**Comment by petesfrench** [comment]
@mtruj013 This is totally a mistake on my part, I was testing it with the wrong form. I'm sorry!!

File: templates/download/mediatek/contact-us.html (hunk 2 above)
Comment (petesfrench) [reply templates/download/mediatek/contact-us.html:109]: I have updated it so the whole form is contained withing the same row

**Comment by petesfrench** [comment]
> @petesfrench is this a known issue and should it be part of this PR? 89RLh0Y6MW [ 89RLh0Y6MW ](https://user-images.githubusercontent.com/2707508/221913347-eee4c8f5-d9d2-49fe-a190-4849f33c36c5.gif) [ ](https://user-images.githubusercontent.com/2707508/221913347-eee4c8f5-d9d2-49fe-a190-4849f33c36c5.gif)

I didn't know about this issue and it is happening in a lot of different place. I did a little digging but it is related to the native behavior of intlTelInput. I have made a [seperate issue](https://github.com/canonical/ubuntu.com/issues/12625) for this as I don't want to block this work or make this PR any more bloated.

**Comment by mtruj013** [comment]
> @mtruj013 This is totally a mistake on my part, I was testing it with the wrong form. I'm sorry!!

No worries at all! Ideally it should be taken care of in dynamic forms, totally agree with you. I remember there being a reason why adding it there didn't work at the time but I can't remember what it was for the life of me. Maybe it's not relevant anymore? Anyway, that's probs out of scope for this pr, there's more than one form with the same issue 😅

**Comment by petesfrench** [comment]
@mtruj013 That's what I thought also. I had a quick look at dynamic-forms.js and it looks like it builds the comments_from_lead message when you update the form in some way (ie. opening it or changing page) but not when you submit. So if there are custom fields on the same page as the submit button, it wont capture those fields. I will leave it for now as your solution works great and address this in the dynamic-form.js refactor

**Review by tbille — APPROVED** [review]
Looks good.
~~Did you manage to add the code for radio buttons that have an input field (like `- [ ] Others: [          ]`) ?~~ Just read the QA section.

**Pull request merged**